src/Game.java

```diff
@@ -0,0 +1,32 @@
+public class Game {
+
+    Players playerOne;
+    Players playerTwo;
+
+    public Game(Players firstPlayerName, Players secondPlayerName) {
```

Comment:
Why is it called `xxxName`?

Reply:
It makes it easier for me to understand what the __constructor__ will *accept*

Reply:
But the constructor won't accept player **names**. It accepts the whole **player** itself, not only the name.